### PR TITLE
Apply for cloud provider baiducloud.

### PR DESCRIPTION
**Pull request by tizhou86**
Hi Kim, this pr is for applying for cloud provider baiducloud in Kubernetes community, please make a review and let me know if you have any questions, thanks! @andrewsykim

### CODE REVIEW

**Comment by andrewsykim** [comment]
Thanks @tizhou86 can you also update https://github.com/kubernetes/community/blob/master/sigs.yaml#L602-L618?

File: keps/sig-cloud-provider/providers/cloud-provider-baiducloud.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+- Designing, discussing, and maintaining the cloud-provider-baidu repository under Github Kubernetes project. 
+
+### Non-Goals
+
+- Identify domain knowledge and work that can be contributed back to Kubernetes and related CNCF projects.
+
+- Mentor CCE and BPC developers to contribute to CNCF projects.
+
+- Provide an open platform for Baidu Kubernetes community to discuss Kubernetes related technologies and share knowledge.
```
Comment (andrewsykim) [inline keps/sig-cloud-provider/providers/cloud-provider-baiducloud.md]: Was this supposed to be in `Goals` instead of `Non-Goals`?

Comment (tizhou86) [reply keps/sig-cloud-provider/providers/cloud-provider-baiducloud.md]: Revised.

**Comment by andrewsykim** [comment]
FYI this KEP is still in the `provisional` status, see https://github.com/kubernetes/community/pull/2449

**Pull request merged**